plugin.yaml

```diff
@@ -1,8 +1,8 @@
 name: "migrate-values"
-version: "0.0.1"
```

Comment:
this was because the version in plugin.yaml was never being updated in accordance with the git tags